default/python/universe_generation/universe_tables.py

```diff
@@ -64,85 +64,86 @@
 # none will be forced to have at least one satellite.
 STAR_TYPE_MOD_TO_PLANET_SIZE_DIST = {
     #                           none  tiny  small  medium  large  huge  asteroids  gas giant
-    fo.starType.blue:      (0,  0, 0,  0, -5,  -10, 0,   0),
```

Comment:
all these whitespace changes being done in the previous commit and then undone in this commit is a lot of noise. can you just remove it from the pull request entirely?

Reply:
i'll try.

Reply:
done.